src/bigquery.ts

```diff
@@ -1139,6 +1146,12 @@ export class BigQuery extends common.Service {
     callback: DatasetCallback
   ): void;
   createDataset(id: string, callback: DatasetCallback): void;
+  /**
+   * @callback DatasetCallback
+   * @param {?Error} err Request error, if any.
+   * @param {object} dataset The [dataset resource]{@link https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets#resource}.
+   * @param {object} apiResponse The full API response.
+   */
```

Comment:
I realize this is probably just a node-ism, but this idea of comment blocks one after another that aren't anchored to something outside the comment block weirds me out. :)

Reply:
I think that it is a TypeScript-ism because it looks like the pattern occurs in some samples [here](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#typedef-callback-and-param).